src/frontend-node-types.ts

```diff
@@ -383,6 +385,14 @@ export type FrontendVideoNode = EditorVideoPlugin & {
   pluginId?: string
 }
 
+export type FrontendAudioNode = EditorAudioPlugin & {
+  type: FrontendNodeType.Audio
+  children?: undefined
+  pluginId?: string
+  src: string
+  base64AudioRecording: string
```

Comment:
You store the audio base64 encoded? -> I would rather want to upload it to our media store instead of storing it into our mysql...

Reply:
Would vocaroo support be enough for a prototype?

We have to remember that storing data in a revision means that the data is duplicated on every edit of the content, is also puts more pressure on the redis cache (which is already struggling ..)

Reply:
@CodingDive it's a neat feature to record audio directly on serlo but I'll agree with @Entkenntnis that is the part that is out of scope of a prototype like this.